src/main.cc

```diff
@@ -253,6 +253,7 @@ int main(int argc, char *argv[])
         exitCode = app->exec();
     }
 
+    app->_shutdown();
```

Comment:
The only real risk that I can see is objects living on other threads may still be alive and running at this point, emitting signals and interacting with objects that lived on the main thread. It shouldn't be a huge deal since event processing is dead by this point (so signals would never reach their respective slots).